doc/windows/package-manager/winget/upgrade.md

```diff
@@ -24,20 +24,38 @@ The options allow you to customize the upgrade experience to meet your needs.
 
 | Option | Description |
 |-------------|-------------|  
-| **-m, --manifest** | Must be followed by the path to the manifest (YAML) file. You can use the manifest to run the upgrade experience from a [local YAML file](#local-upgrade). |
-| **--id** |  Limits the upgrade to the ID of the application.   |  
-| **--name** |  Limits the search to the name of the application. |  
-| **--moniker** | Limits the search to the moniker listed for the application. |  
-| **-v, --version** |  Enables you to specify an exact version to upgrade. If not specified, latest will upgrade the highest versioned application. |  
-| **-s, --source** |  Restricts the search to the source name provided. Must be followed by the source name. |  
-| **-e, --exact** | Uses the exact string in the query, including checking for case-sensitivity. It will not use the default behavior of a substring. |  
-| **-i, --interactive** | Runs the installer in interactive mode. The default experience shows installer progress. |  
-| **-h, --silent** | Runs the installer in silent mode. This suppresses all UI. The default experience shows installer progress. |  
-| **-o, --log** |  Directs the logging to a log file. You must provide a path to a file that you have the write rights to. |
```

Comment:
I think the original descriptions here contain important detail. Anyone can use the `-?` to see the command information, but that doesn't tell anything about it needing to be a path that you have write access to, which is a very important detail that someone looking at documentation would probably expect.

Reply:
Reverted to original descriptions to the commands that had one